strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/ClassDependenciesGraph.java

```diff
@@ -76,6 +76,10 @@ void buildTestDependencyGraph(Collection<File> testJavaFiles) {
                 addToIndex(new JavaElement(javaClass), javaClass.getImports());
             }
         }
+
+        if (Boolean.valueOf(System.getProperty("smart.testing.debug", "false"))) {
```

Comment:
The default value is not necessary here - from the javadoc of `Boolean.valueof`
```
    /**
     * Returns a {@code Boolean} with a value represented by the
     * specified string.  The {@code Boolean} returned represents a
     * true value if the string argument is not {@code null}
     * and is equal, ignoring case, to the string {@code "true"}.
     *
     * @param   s   a string.
     * @return  the {@code Boolean} value represented by the string.
     */
```

Reply:
It will be gone anyway when we merge with debug mode, but good catch.

Reply:
It's a bit more than that - see https://github.com/arquillian/smart-testing/issues/113#issuecomment-329296655